.github/workflows/format.yml

```diff
@@ -0,0 +1,23 @@
+name: Code Format Checks
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
should this be `master` or `main`?

Reply:
Nice catch! c159015c54c56a1a4c7ba855b7dbceee0833dfdd